manticore/platforms/evm.py

```diff
@@ -1884,16 +1896,14 @@ def __str__(self):
         def hexdump(src, length=16):
             FILTER = ''.join([(len(repr(chr(x))) == 3) and chr(x) or '.' for x in range(256)])
```

Comment:
brackets `[...]` are redundant (as in https://github.com/trailofbits/manticore/pull/784/files?diff=unified#r175897327)